UefiCpuPkg/Library/CpuExceptionHandlerLib/SecPeiCpuExceptionHandlerLibMu.inf

```diff
@@ -49,10 +49,14 @@
   PrintLib
   LocalApicLib
   PeCoffGetEntryPointLib
-  VmgExitLib
+  CcExitLib
   ResetSystemLib
   ExceptionPersistenceLib
 
+[Pcd]
+  gUefiCpuPkgTokenSpaceGuid.PcdCpuStackSwitchExceptionList
+  gUefiCpuPkgTokenSpaceGuid.PcdCpuKnownGoodStackSize

```

Comment:
We can merge this commit into the one adding the lib.

Reply:
Done.